dependencyManagement/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val TEST_SNAPSHOTS = rootProject.findProperty("testUpstreamSnapshots") == "true"
 
 // This is the version of the upstream instrumentation BOM
-val otelVersion = "1.25.1"
+val otelVersion = "1.25.1-adot1"
```

Comment:
Why `adot1` and not `adot`?

Reply:
because it is in the docs https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/RELEASING.md#patching-upstream-dependencies

Reply:
>The adot patch version of each upstream dependency should be <version>-adot<number> where version is the version of the upstream dependency and number is the number of this patch that should be incremented from 1 per patch version.

Reply:
the number is necessary in case we need to apply patch on top of other patches.